crates/iceberg/src/avro/schema.rs

```diff
@@ -30,6 +32,11 @@ use itertools::{Either, Itertools};
 use serde_json::{Number, Value};
 
 const FILED_ID_PROP: &str = "field-id";
+const UUID_BYTES: usize = 16;
+const UUID_LOGICAL_TYPE: &str = "uuid";
+// # TODO
+// This const may better to maintain in avro-rs.
```

Comment:
We can create an issue for avro-rs and comment the issue link here.

Reply:
Have added a #86 to track them